flocker/acceptance/endtoend/test_installer.py

```diff
@@ -129,19 +129,21 @@ def wait_for_stack_status(stack_id, target_status):
     :param unicode target_status: The desired stack status.
     :returns: A ``Deferred`` which fires when the stack has ``target_status``.
     """
-    Message.log(
-        function='wait_for_stack_status',
-        stack_id=stack_id,
-        target_status=target_status,
-    )
-
     def predicate():
         stack_report = get_stack_report(stack_id)
         current_status = stack_report['StackStatus']
+        Message.log(
+            function='wait_for_stack_status',
+            stack_id=stack_id,
+            target_status=target_status,
+            current_status=current_status
+        )
         if current_status == target_status:
             return stack_report
 
-    return loop_until(reactor, predicate, repeat(10, 60))
+    return loop_until(reactor,
+                      predicate,
+                      repeat(10, 120))
```

Comment:
:question: This means wait for 20min, right? I guess we could just let this loop forever and let the 30 minute test timeout do its thing if the stack creation + tests take longer. We can decide on that when we see how long Jenkins takes to run the tests.